packages/chat/src/caps/twitchCommands/messageTypes/ClearChat.ts

```diff
@@ -14,4 +14,12 @@ export class ClearChat extends Message<ClearChat> {
 		optional: true
 	})
 	user!: MessageParam;
+
+	get channelId(): string {
+		return this._tags.get('room-id')!;
+	}
+
+	get targetUserId(): string {
+		return this._tags.get('target-user-id')!;
```

Comment:
This isn't always set by Twitch (when clearing the whole chat, no user is targeted), so you shouldn't non-null-assert it.

Reply:
This was my thought while [writing this](https://github.com/twurple/twurple/pull/309#discussion_r745250336)

Reply:
`onTypedMessage` is a public method, so messages can be accessed by anyone, and then it can definitely be null here.